WordPress/Classes/ViewRelated/Reader/ReaderPostCellActions.swift

```diff
@@ -88,11 +89,23 @@ class ReaderPostCellActions: NSObject, ReaderPostCellDelegate {
     }
 
     func readerCell(_ cell: ReaderPostCardCell, menuActionForProvider provider: ReaderPostContentProvider, fromView sender: UIView) {
-        guard let post = provider as? ReaderPost, let origin = origin else {
+        guard let post = provider as? ReaderPost,
+              let origin = origin,
+              let _followCommentsService = FollowCommentsService(post: post) else {
```

Comment:
Minor comment: how about using `followCommentsService` as the variable name instead? I'm not sure if we have this defined in our style guide, but in this case, we'd prefer to keep `followCommentsService` for the local variable and use `self.followCommentsService` to refer to the instance variable.

Reply:
Hey you're right! Accessing the `followCommentsService` then takes the local variable. No need for underscore workarounds. I'll update it.